### PR TITLE
shadow on tile hover

### PR DESCRIPTION
**Pull request by diomed**
also updated bulma version to 0.6.2.

### CODE REVIEW

File: css/hero.css [hunk 1]
```diff
@@ -50,4 +50,35 @@ html,body {
...
+.is-shady:nth-child(2) {
+animation-delay: .45s
+}
+.is-shady:nth-child(3) {
+animation-delay: .7s
+}
+.is-shady:nth-child(4) {
+animation-delay: .35s
+}
+.is-shady:nth-child(5) {
```
Comment (dansup) [inline css/hero.css]: This assumes a 5th child, while I appreciate the effort, my goal is to keep the custom css to a minimum to showcase the bulma framework so the templates are easier to customize. Can you remove the shadows?

Comment (diomed) [reply css/hero.css]: whole point of this pr is in shadows. I dont understand why bulma doesnt have them, and would like to give this ability to templates.

Comment (dansup) [reply css/hero.css]: @diomed Adding box-shadow is one thing, but you have to remove the animation stuff. A ```.box-shadow``` class with a hover attribute would work, and I have no problem with adding a single class but everything else has to be removed.

**Comment by diomed** [comment]
⁉️

**Comment by dansup** [comment]
@diomed Thanks for the PR!

**Pull request merged**